src/components/flexjar/flexjar-felles.tsx

```diff
@@ -100,12 +102,13 @@ export function FlexjarFelles({
             setActiveState(null)
             setTextValue('')
             setThanksFeedback(true)
+            return
         }
     }
 
     return (
-        <section>
-            <div className="w:full mt-16 md:w-3/4">
+        <section className="w-full mt-16">
```

Comment:
hvis vi permanent endrer alle til w-full så må vi også oppdatere de to andre appene

Reply:
alternativt kan vi jo med en prop sette den til w-full når vi trenger det, aka fra flexjar-survey